checker/jdk/determinism/src/java/io/PrintStream.java

```diff
@@ -58,7 +58,7 @@
  */
 
 public class PrintStream extends FilterOutputStream
-    implements Appendable, Closeable
```

Comment:
You've only annotated public methods in this class.  Please a comment stating that.  You should probably also annotate protect methods too since a subclass could call those.